packages/react/src/components/Tooltip/index.tsx

```diff
@@ -44,15 +44,14 @@ export default function Tooltip({
   target,
   association = 'aria-describedby',
   variant = 'text',
-  show: showProp = false,
+  show: initialShow = false,
   hideElementOnHidden = false,
   className,
   ...props
 }: TooltipProps) {
   const [id] = propId ? [propId] : useId(1, 'tooltip');
-  const hovering = useRef(false);
-  const [placement, setPlacement] = useState(initialPlacement);
```

Comment:
We don't need to store this as state. Doing so was basically a no-op: https://github.com/dequelabs/cauldron/blob/develop/packages/react/src/components/Tooltip/index.tsx#L115-L123